resolwe/test_helpers/management/commands/show_profile.py

```diff
@@ -0,0 +1,68 @@
+""".. Ignore pydocstyle D400.
```

Comment:
Hmm, it appears that one cannot use the newly added management commands from Resolwe's test Django project.

I think it would be useful to add `resolwe.test_helpers` (as I suggested in the other comment, I would actually move the management commands under `resolwe.test`) to `INSTALLED_APPS` so one can use these commands.

Reply:
They are not in Resolwe's test project, because they are not used in Resolwe's tests. They are used for example in Resolwe Bio's tests.

Reply:
Ok. I thought it was not left out on purpose.